.github/workflows/tugboat.yml

```diff
@@ -48,11 +48,14 @@ jobs:
               target_url: "https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}"
             });
 
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
           node-version: '14'
 
+      - name: yarn install
+        run: yarn workspace e2e-test install
```

Comment:
i'm curious, does the workspace scope make a difference here, when it's in a lerna monorepo?

Reply:
Nop, better use our cached yarn install steps instead